eos-payg-csv/README

```diff
@@ -0,0 +1,29 @@
+eos-payg-csv
+============
+Given a specially-formatted .csv file (see below) containing device IDs and
+private keys, generate .csv files containing the key-specific time codes for a
+limited set of time periods.
+
+File format
+===========
+The expected format of this file is a CSV with the following header row 
+and one or more device rows:
+
+header row
+----------
+device_id,code 1, code 2, code 3,key
+
+device row
+----------
+DEVICE_ID,CODE_1,CODE_2,CODE_3,"KEY"
+
+where:
```

Comment:
Would be useful to provide a sample CSV file in the README, to illustrate the description.